helm/prometheus-rules/templates/alerting-rules/loki.rules.yml

```diff
@@ -73,7 +73,8 @@ spec:
         description: '{{`Loki pod {{ $labels.pod }} (namespace {{ $labels.namespace }}) sees {{ $value }} unhealthy ring members`}}'
         opsrecipe: loki/
       expr: |
-        sum (min_over_time(cortex_ring_members{state="Unhealthy"}[30m])) by (app, cluster_id, container, customer, installation, provider, pipeline, name, namespace, organization, pod) > 0
```

Comment:
Do WE want this to page only when app=Loki?

Reply:
You mean with Mimir, the query can match mimir pods ?

Reply:
I think so yes